depthai_ros_driver/include/depthai_ros_driver/device_ros.hpp

```diff
@@ -1,21 +1,30 @@
-#include <depthai_ros_driver/dai_utils.hpp>
+// package headers
 #include <depthai_ros_driver/conversion.hpp>
+#include <depthai_ros_driver/dai_utils.hpp>
 
+// relevant 3rd party headers
 #include <depthai/device/DeviceBase.hpp>
 #include <depthai/pipeline/datatype/StreamPacketParser.hpp>
 #include <depthai/pipeline/node/XLinkOut.hpp>
 #include <depthai/pipeline/node/XLinkIn.hpp>
 
+// ros headers
+#include <camera_info_manager/camera_info_manager.h>
+#include <ros/callback_queue.h>
 #include <ros/publisher.h>
 #include <ros/ros.h>
-#include <ros/callback_queue.h>
 
-#include <range/v3/view.hpp>
+// lib headers
 #include <range/v3/algorithm/find.hpp>
+#include <range/v3/view.hpp>
+
+// std headers
```

Comment:
Do we have any convention for order of the include?

Google, [for example](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) include C header, C++ header, 3rd party header, followed by project header.

Reply:
I use increasing order of trust of no error:
* project headers
* other library headers
* famous library headers
* cpp headers
* c headers

This ensures that bad includes are detected asap. Ofc, if the includes have some constraint on the order due to macros, nothing can be done